src/components/organisms/Modal.js

```diff
@@ -7,26 +7,30 @@ const Modal = ({
   title,
   subTitle,
   children,
+  buttons,
   submitHandler,
   cancelHandler,
   confirmText,
   cancelText,
-  canCancel,
 }) => {
-  const handleSubmit = (e) => {
-    e.preventDefault();
-    submitHandler(e);
-  };
-
-  const handleCancel = () => {
-    cancelHandler();
+  const handleCancel = (event) => {
+    if (!cancelHandler) {
+      return;
+    }
+    cancelHandler(event);
   };
 
   return (
     <div
       className="modal"
     >
-      <form onSubmit={handleSubmit}>
+      <form
+        onSubmit={
+          submitHandler
+            ? (event) => submitHandler(event)
+            : (event) => event.preventDefault()
+        }
+      >
```

Comment:
We're now checking for a submitHandler, if none's found we preventdefault and exit out